packages/js/email-editor/src/editor.tsx

```diff
@@ -42,5 +47,6 @@ export function initialize( elementId: string ) {
 	initBlocks();
 	initHooks();
 	const root = createRoot( container );
-	root.render( <Editor /> );
+	// @ts-ignore
```

Comment:
It would be good to add a comment on why we need to use `@ts-ignore`. Please, can you add it?

Reply:
Noted. Done in https://github.com/mailpoet/mailpoet/pull/5939/commits/3f74dacb51aef9724547a819098f911c4a58c502